src/modules/live-execution-tracker/services/live-execution-tracker.service.ts

```diff
@@ -179,6 +179,19 @@ export class LiveExecutionTrackerService implements ILiveExecutionTrackerService
     return elementsWithTokenHistory;
   }
 
+  private async getElementsWhereProcessWasStopped(processInstanceId: string): Promise<Array<IShape>> {
```

Comment:
Sorry, missed that one:

```suggestion
  private async getElementsWithError(processInstanceId: string): Promise<Array<IShape>> {
```